content/integrations/awscodedeploy.md

```diff
@@ -4,6 +4,7 @@ integration_title: AWS CodeDeploy
 kind: integration
 git_integration_title: amazon_codedeploy
 newhlevel: true
+description: "See deployments as they happen and track how long they take."
```

Comment:
"See AWS CodeDeploy deployments  in your event stream as they happen,  as well as track completion times and success rates."

Reply:
👍